macOS/LocalPackages/NewTabPage/Sources/NewTabPage/Configuration/NewTabPageConfigurationClient.swift

```diff
@@ -106,34 +95,20 @@ public final class NewTabPageConfigurationClient: NewTabPageUserScriptClient {
     }
 
     private func fetchWidgets() -> [NewTabPageDataModel.NewTabPageConfiguration.Widget] {
-        var widgets: [NewTabPageDataModel.NewTabPageConfiguration.Widget] = [
+        [
             .init(id: .rmf),
             .init(id: .freemiumPIRBanner),
             .init(id: .nextSteps),
             .init(id: .favorites),
+            .init(id: .protections)
```

Comment:
Sections availability provider is not needed anymore because Protections Report widget is always available for all users.